clients/src/main/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumer.java

```diff
@@ -1501,11 +1501,11 @@ public void unsubscribe() {
 
     private void resetGroupMetadata() {
         groupMetadata.updateAndGet(
-            oldGroupMetadataOptional -> oldGroupMetadataOptional
-                .map(oldGroupMetadata -> initializeConsumerGroupMetadata(
-                    oldGroupMetadata.groupId(),
-                    oldGroupMetadata.groupInstanceId()
-                ))
+                oldGroupMetadataOptional -> oldGroupMetadataOptional
+                        .map(oldGroupMetadata -> initializeConsumerGroupMetadata(
+                                oldGroupMetadata.groupId(),
+                                oldGroupMetadata.groupInstanceId()
+                        ))
```

Comment:
format is off here right? (I guess it's because you had it inside an `if` at some point, but was then removed). I expect we should end up with the `resetGroupMetadata` unchanged in this PR

Reply:
Yeah, we should revert this change. Thanks for the comment. I have rebased trunk branch.